src/config.rs

```diff
@@ -209,7 +214,10 @@ pub enum Config {
     /// Configured IMAP server security (e.g. TLS, STARTTLS).
     ConfiguredMailSecurity,
 
-    /// How to check IMAP server TLS certificates.
+    /// How to check TLS certificates.
+    ///
+    /// This actually applies to both IMAP and SMTP connections,
+    /// but has "IMAP" in the name for backwards compatibility.
     ConfiguredImapCertificateChecks,
```

Comment:
I think the difference between `ImapCertificateChecks` and `ConfiguredImapCertificateChecks` should be documented, currently it's not clear from the doc comments